src/FormNetworks/bilinearformnetwork.jl

```diff
@@ -0,0 +1,62 @@
+default_bra_vertex_map(v) = (v, "bra")
+default_ket_vertex_map(v) = (v, "ket")
+default_operator_vertex_map(v) = (v, "operator")
+default_operator_constructor(s::IndsNetwork) = delta_network(s)
+
+struct BilinearFormNetwork{V,KetMap,BraMap,OperatorMap} <: AbstractFormNetwork{V}
+  tn::AbstractITensorNetwork{V}
+  bra_vertex_map::BraMap
+  ket_vertex_map::KetMap
+  operator_vertex_map::OperatorMap
```

Comment:
```suggestion
  operator_vertex_map::OperatorMap
  bra_vertex_map::BraMap
  ket_vertex_map::KetMap
```
Let's order the fields in the ordering of the standard constructor call.

Reply:
Looks like the field ordering is still bra-ket-operator instead of operator-bra-ket.

Reply:
Huh, convinced I fixed this. Anyway, should be fixed now.